finality-provider/service/event_loops.go

```diff
@@ -189,6 +182,11 @@ func (app *FinalityProviderApp) unjailFpLoop() {
 				zap.String("txHash", res.TxHash),
 			)
 
+			// set the status to INACTIVE by default
```

Comment:
This is for the case where the fp was jailed, stoped the fpd, unjail in Babylon and start fpd again?

Reply:
I think the point of this pr is to allow the fp to be running continously running even though it got jailed and unjailed in the middle